app/templates/signed-out.html

```diff
@@ -1,6 +1,6 @@
 {% extends theme('layouts/_twocol.html') %}
 
-{% block page_title %}Save Successful{% endblock %}
```

Comment:
Where there isn't a survey title the base page could fallback to a safe default using a filter/if?